build-tools-internal/build.gradle

```diff
@@ -291,6 +291,8 @@ dependencies {
   api buildLibs.httpclient
   api buildLibs.httpcore
 
+  implementation 'com.github.javaparser:javaparser-core:3.18.0'
```

Comment:
they have build-in receipts for stuff like this I think. IMO, Ideally we should use openrewrite here as its actually made for exact this type of automatic code updates. I will have a look if openrewrite is actually easier than this little java-parser logic we have here and how maintanable it is.

Reply:
Doing this sort of modification is more difficult in openrewrite's visitor pattern - it lends itself a lot better to bulk-modifying existing code in a pattern, rather than specific method changes like here